### PR TITLE
Added projects link to the nav bar.

### PR DESCRIPTION
**Pull request by Divy123**
Resolved issue #167

### CODE REVIEW

**Comment by pradeepgangwar** [comment]
@Divy123 Again saying please link the issue in the description. Else won't be reviewed.

**Comment by pradeepgangwar** [comment]
Live link?

**Comment by pradeepgangwar** [comment]
@Divy123 Are you fixing it?

**Comment by Divy123** [comment]
@pradeepgangwar I am not getting it...
Can you plezase tell how to do this?

**Comment by Divy123** [comment]
Resolved issue #167 
Live link
https://divy123.github.io/OpenCode-Collaborative/

File: index.html [hunk 1]
```diff
@@ -76,6 +76,9 @@
           </li>
           <li class="nav-item">
             <a class="nav-link js-scroll-trigger" href="#contact">Participants</a>
+          </li>
+		<li class="nav-item">
+            <a class="nav-link js-scroll-trigger" href="https://opencode18.github.io/projects">Projects</a>
```
Comment (pradeepgangwar) [inline index.html]: Link href should be #projects . As we have project section in website now.

**Comment by pradeepgangwar** [comment]
Resolve merge conflicts else is fine.

**Comment by pradeepgangwar** [comment]
@Divy123 I don't know who added that project section on nav bar. Please leave this PR here. I will see if points can be allocated to you for your work.

**Comment by pradeepgangwar** [comment]
Points will be allocated to @Divy123 for his time and work. Thanks @Divy123

**Comment by Divy123** [comment]
@pradeepgangwar is that someone else who solved before me in pr?

**Comment by pradeepgangwar** [comment]
@Divy123 Its hard to keep track of that's why we always ask to reference issue. I don't know who did that. Next time it won't happen.

**Comment by Divy123** [comment]
Ok...

**Pull request closed without merging**